arbos/multigasCollector/proto/multigas.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package multigas.proto;
+
+option go_package = "github.com/ethereum/go-ethereum/arbitrum/multigas/proto";
+
+// Multi-dimensional gas consumption data mirroring multigas.MultiGas struct
+message MultiGasData {
+  uint64 computation = 1;
```

Comment:
I know I'm going to sound crazy here. But, I really prefer to make all the fields on a proto optional unless we're 100% sure that we would NEVER remove them.

Right now, if we later decide to remove the concept of history_growth (for example), we'd have trouble omitting the field, because the value is required to construct a valid MultiGasData.

I usually only consider something like "id" fields for making them required (non-optional). And, I don't think most of these protos have such a field. Maybe `tx_index` or `block_number`. But, I'd make everything else `optional`